Topics/Topic_2_b/src/main/java/com/bootcamp/topic2_b/blog/BlogList.java

```diff
@@ -0,0 +1,81 @@
+package com.bootcamp.topic2_b.blog;
+
+import java.util.Iterator;
+
+public class BlogList implements Iterable<Entry> {
```

Comment:
I tried to do a List, because we agree that in our last meeting.
With that... I supposed I was doing the right thing...